src/main/java/org/opentripplanner/middleware/utils/HttpUtils.java

```diff
@@ -70,7 +77,6 @@ public static String httpRequest(URI uri, int connectionTimeout, HttpMethod meth
      */
     public static HttpResponse<String> httpRequestRawResponse(URI uri, int connectionTimeout, HttpMethod method,
```

Comment:
I did some searching in the code and found one another (pretty insignificant) use of HttpClient in `PublicApiDocGenerator.class`. I think we should replace that use of `client` in `getSwaggerDocs` with:

```java
return HttpUtils.httpRequestRawResponse(
    URI.create(AUTOGENERATED_DOC_URL), 
    10, 
    HttpMethod.GET, 
    null, 
    null
);
```

This is mainly just to reinforce the use of this util file throughout the app.

Also, one other minor thing: could you change the `connectionTimeout` param to `timeoutInSeconds`?

Reply:
Updated.